src/data/grammar.json

```diff
@@ -62,7 +62,7 @@
         "#titleMoney#: #investType# #card# #investReason#",
         "#titleMoney#: #investType# #card#",
         "#titleMoney#: #pro# is #investType#ing #broadCardClassification.capitalizeAll#; Should You?",
-        "BREAKING: Prices of #card.capitalizeAll# Spiking; Market Cornered by #pros#",
```

Comment:
On second thought, I think the cards should be responsible for their own capitalization.